nomad/api/base.py

```diff
@@ -17,9 +18,17 @@ def _endpointBuilder(self, *args):
             return "{v}/".format(v=self.version) + u
 
     def _urlBuilder(self, endpoint):
-        return "http://{uri}:{port}/{endpoint}".format(uri=self.uri,
-                                                       port=self.port,
-                                                       endpoint=endpoint)
+        if self.verify:
+            proto = 'https'
+        else:
+            proto = 'http'
+
+        return "{proto}://{uri}:{port}/{endpoint}".format(
+            proto=proto,
+            uri=self.uri,
+            port=self.port,
+            verify=self.verify,
```

Comment:
I think so... because in my case, I will pass in the cert. For example,
`nomad = nomad.Nomad(nomad_server, timeout=2, verify=cert)`

otherwise, the underlying requests library will complain about missing cert or something related to that.